### ISSUE TITLE
nicer releases

### ISSUE THREAD

**Issue by ImLunaHey** [staylor/react-helmet-async#227]
any chance a changelog and/or github releases could be used?

**Comment by arthurdenner** [comment]
I agree, it'd be great to know what changed between versions, especially since there is a v2 now.
For now, if one is curious, here is a link comparing the commit from 1.3.0 to the latest 2.0.5:

https://github.com/staylor/react-helmet-async/compare/5f4da3896c8b70bedcd3c05d406ff5241601933c..b1ee65880e71df9124c4e01d07c086d8ce2fc63a

**Comment by MorrisonCole** [comment]
Also the latest release is `2.0.5` but the main branch is still `2.0.4`... https://github.com/staylor/react-helmet-async/blob/main/package.json#L3

**Comment by xander-marjoram** [comment]
@arthurdenner Thanks for the link.

@staylor would it be possible to summarise the breaking changes between v1.3.0 and v2.0.0?

**Comment by ImLunaHey** [comment]
@staylor it's been almost a year with no replies and no updates in this repo. is this still being maintained?
